### PR TITLE
data: utkarshkrishna2004

### PR DESCRIPTION
**Pull request by utkarshkrishna2004**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5642"></a>

### CODE REVIEW

File: data/utkarshkrishna2004.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Utkarsh Krishna",
+  "type": "personal",
+  "bio": "Open Source enthusiast | Aspiring Full-Stack Developer | Writing Tech Blogs",
+  "avatar": "https://github.com/utkarshkrishna2004.png",
```
Comment (Panquesito7) [inline data/utkarshkrishna2004.json:5]: Your avatar is automatically fetched from your GitHub profile.
```suggestion
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by khushalarora — APPROVED** [review]
Looks good :tada:

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by utkarshkrishna2004** [comment]
Hey people, so I have a question, if I do some changes(say, I want to add my Instagram profile too), do I again have to create a PR for those changes to reflect in my LinkFree page?

**Pull request merged**